lib/modules/platform/azure/azure-helper.ts

```diff
@@ -141,7 +141,9 @@ export async function getMergeMethod(
   };
 
   const policyConfigurations = (
-    await (await azureApi.policyApi()).getPolicyConfigurations(project)
+    await (
+      await azureApi.policyApi()
+    ).getPolicyConfigurations(project, mergePolicyGuid)
   )
     .filter(
```

Comment:
small nit, but is the filter below still required?

Reply:
no, good point - removed.